src/Entity/Item.php

```diff
@@ -319,4 +319,16 @@ public function equals( EntityDocument $entity ) {
 		       && $this->statements->equals( $entity->statements );
 	}
 
+	/**
+	 * @see EntityDocument::__clone
+	 *
+	 * @since 5.0
+	 */
+	public function __clone() {
+		$this->fingerprint = clone $this->fingerprint;
+		// SiteLinkList is mutable, but SiteLink is not. No deeper cloning necesarry.
```

Comment:
I don't see how this comment helps in this place since you wouldn't do deeper cloning here anyways. Better move it somewhere into `SiteLinkList`.